From 969bcca33828edf53ca4337430f6bf3d2695ce89 Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Mon, 13 Jan 2020 09:34:46 +0800
Subject: [PATCH 17/23] omx: avcenc: scx15: Ignore
 OMX_IndexParamVideoIntraRefresh

Not supported but should still be fine if it is ignored

Change-Id: I095ede37d989461a62e5f02f3a3f51e478b89b35
Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
---
 omx_components/video/avc_sprd/scx15/enc/SPRDAVCEncoder.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/omx_components/video/avc_sprd/scx15/enc/SPRDAVCEncoder.cpp b/omx_components/video/avc_sprd/scx15/enc/SPRDAVCEncoder.cpp
index 0daac06..0489705 100644
--- a/omx_components/video/avc_sprd/scx15/enc/SPRDAVCEncoder.cpp
+++ b/omx_components/video/avc_sprd/scx15/enc/SPRDAVCEncoder.cpp
@@ -997,6 +997,11 @@ OMX_ERRORTYPE SPRDAVCEncoder::internalSetParameter(
         return OMX_ErrorNone;
     }
 
+    case OMX_IndexParamVideoIntraRefresh:
+    {
+        return OMX_ErrorNone;   ///hw encoder may not support this mode
+    }
+
     default:
         return SprdSimpleOMXComponent::internalSetParameter(index, params);
     }
-- 
2.24.0

